pyproject.toml

```
@@ -6,76 +6,103 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [project]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "License :: OSI Approved :: MIT License",
+]
 name = "nomad-measurements"
 dynamic = ["version"]
 description = "A plugin for NOMAD containing base sections for material processing."
 readme = "README.md"
+requires-python = ">=3.9"
 authors = [
     { name = "Hampus Näsström", email = 'hampus.naesstroem@physik.hu-berlin.de' },
     { name = "Andrea Albino", email = 'andrea.albino@physik.hu-berlin.de' },
     { name = "Sebastian Brückner", email = 'sebastian.brueckner@ikz-berlin.de' },
     { name = "Sarthak Kapoor", email = 'sarthak.kapoor@physik.hu-berlin.de' },
 ]
-requires-python = ">=3.9"
-classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.9",
-    "License :: OSI Approved :: Apache Software License",
-]
+license = { file = "LICENSE" }
 dependencies = [
-    "nomad-lab>=1.3.3.dev86",
+    "nomad-lab>=1.3.4",
     "xmltodict==0.13.0",
-    "fairmat-readers-xrd~=0.0.3"
+    "fairmat-readers-xrd~=0.0.3",
+    'pynxtools@git+https://github.com/FAIRmat-NFDI/pynxtools.git@master',
```

@aalbino2 did you check if this works? 
```suggestion
    "pynxtools>=0.6.0",
```